src/components/formStore/table/FormElementTableCell.tsx

```diff
@@ -172,25 +178,26 @@ function FormElementTableCell({ formElement, submission, allowCopy }: Props) {
         formElement.type === 'checkboxes' ||
         (formElement.type === 'select' && formElement.multi)
       ) {
-        const value = unknown as string
-        if (!Array.isArray(value) || !value.length) {
+        if (!Array.isArray(unknown) || !unknown.length) {
```

Comment:
Would repeatableSet data still pass this check?

Reply:
checking now